src/core/lib/iomgr/tcp_custom.h

```diff
@@ -24,6 +24,8 @@
 #include "src/core/lib/iomgr/endpoint.h"
 #include "src/core/lib/iomgr/sockaddr.h"
 
+#define GRPC_CUSTOM_SOCKET_OPT_SO_REUSEPORT (0x00000010u)
```

Comment:
isn't there existing macro in any of the library headers you can use instead of defining a new one?

Reply:
I have searched the code base, and didn't find a good match. There is `GRPC_ARG_ALLOW_REUSEPORT` in `grpc_types.h`, but it is a string macro instead of a flag value. Do you have another macro I can reuse in mind?